common/src/main/java/com/wynntils/mc/EventFactory.java

```diff
@@ -194,6 +211,12 @@ public static void onKeyInput(int key, int scanCode, int action, int modifiers)
         post(new KeyInputEvent(key, scanCode, action, modifiers));
     }
 
+    public static boolean onChatSend(String message) {
```

Comment:
this should support editing the sent message as well; it's important for features such as chat items

maybe this could be accomplished by simply returning the event object?

Reply:
yeah imo `EventFactory` should either return the event or a boolean

Reply:
I think we need further changes in the mixin to be able to edit the message. I'm not against it, but suggest we wait with implementing it until it's needed (so we can notice if it does not work).

Reply:
@P0keDev Are you ok with this?